CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [PEP 440](https://www.python.org/dev/peps/pep-0440/)
 and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 5.1.3
+
+### Security
+
+* It is now possible to inject Earthdata username and password using environment variables.
```

Comment:
I think it's worth calling out the names of the environment variables here, so a casual user won't have to dig through the implementation to find them, but again it's not a sticking point since I'm not sure any such users exist at the moment